libfwupd/fwupd-security-attr.c

```diff
@@ -584,6 +593,46 @@ fwupd_security_attr_set_bios_setting_current_value(FwupdSecurityAttr *self, cons
 	priv->bios_setting_current_value = g_strdup(value);
 }
 
+/**
+ * fwupd_security_attr_get_kernel_cmdline_value:
```

Comment:
I'd like if you had `target` in the name, although I admit it makes a really long function name

Reply:
I dropped target as we didn't really have `current`. Maybe that's the bug -- i.e. maybe we want `lockdown=integrity` as the _current_ value and `lockdown=confidentiality` as the _target_ value?

Reply:
Yes! Then it's a lot more obvious if something is getting corrected what's happening.

Reply:
Fixed